addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -53,7 +53,7 @@ def _l10n_it_edi_generate_electronic_invoice_filename(self, invoice):
 
         return '%(country_code)s%(codice)s_%(progressive_number)s.xml' % {
             'country_code': invoice.company_id.country_id.code,
-            'codice': invoice.company_id.l10n_it_codice_fiscale.replace(' ', ''),
+            'codice': (invoice.company_id.l10n_it_codice_fiscale[2:] if invoice.company_id.l10n_it_codice_fiscale[:2] == 'IT' else invoice.company_id.l10n_it_codice_fiscale).replace(' ', ''),
```

Comment:
For reading purpose, it would be better to put `(invoice.company_id.l10n_it_codice_fiscale[2:] if invoice.company_id.l10n_it_codice_fiscale[:2] == 'IT' else invoice.company_id.l10n_it_codice_fiscale)` in a variable

Reply:
I think you're right, but I found there was a function that does this, which I think is cleaner even cleaner.